x-pack/plugins/log_explorer/common/constants.ts

```diff
@@ -27,12 +29,18 @@ export const LOG_FILE_PATH_FIELD = 'log.file.path';
 export const DATASTREAM_NAMESPACE_FIELD = 'data_stream.namespace';
 export const DATASTREAM_DATASET_FIELD = 'data_stream.dataset';
 
+// Virtual column fields
+export const CONTENT_FIELD = 'content';
+
 // Sizing
 export const DATA_GRID_COLUMN_WIDTH_SMALL = 240;
 export const DATA_GRID_COLUMN_WIDTH_MEDIUM = 320;
 
 // UI preferences
 export const DEFAULT_COLUMNS = [
+  {
+    field: CONTENT_FIELD,
+  },
```

Comment:
TBH, no rationale. So timestamp is always the 1st column. As per the new designs, right after the timestamp we will place Resource Column which is not yet there and then Content Column. So i placed Content as 1st after timestamp. Once we add Resource, that would actually take this spot and Content would move next

Reply:
Right... I stumbled across this, because the current host name and service name columns are sort-of the predecessor for the resource column. So what if we put them in the same location that the resource column will be in in the future?

Reply:
Sounds good